workflows/wf2/workflow.json

```diff
@@ -6,11 +6,20 @@
                 "adaptor": "openmrs",
                 "configuration": "../tmp/openmrs-creds.json",
                 "state": {
-                    "manualCursor": "2023-07-27T07:16:24.544Z"
+                    "lastRunDateTime": "2024-12-20T08:00:00.000Z"
                 },
                 "expression": "1-get-patients.js",
                 "next": {
-                    "upsert-teis": true
+                    "mappings": "!state.errors"
```

Comment:
@mtuchi why is this workflow.json so different than what was currently in the repo? Want to double check before I merge just to make sure there's nothing weird going on here

Reply:
@aleksa-krolls thanks for checking, The changes on this file are replication of what is on app.openfn.org, So it's up to date